docs/requirements.txt

```diff
@@ -0,0 +1,10 @@
+avro
```

Comment:
Why do we need a `requirements.txt`? Shouldn't the `setup.py` already handle this? Is this required for the readthedocs integration or something? I just want to avoid duplicating dependency information if possible.